eia_app/models.py

```diff
@@ -279,3 +279,25 @@ class DatosDocumento(models.Model):
     def get_model_type(self):  # pylint: disable=no-self-use
         '''Devuelve el tipo de modelo'''
         return "Datos_Documento"
+
+
+class DescripcionProyecto(models.Model):
+    """ Tabla para almacenar la informacion de la descripcion de un proyecto.
+    Parametros:
+        models.Model (DescripcionProyecto): Instancia sobre la que se crea la tabla.
+    Atributos:
+        proyecto: Proyecto al que pertenencen los datos del documento.
+        objGeneral: objetivo general del estudio
+        objEspecifico: objetivos especificos del estudio
+        justificacion: azones por las que se realiza el estudio
+        area: mapa del sitio o espacio donde se realiza el estudio
+    """
+    proyecto = models.ForeignKey(DatosProyecto, on_delete=models.CASCADE)
+    objGeneral = models.TextField()
```

Comment:
Igual que con objEspecifico.
```suggestion
    obj_general = models.TextField()
```